README.md

```diff
@@ -2,7 +2,7 @@
 
 To get up and running with scaffold, ensure the dependencies are installed:
 
-    npm run-script install-dependencies
+    npm install
```

Comment:
@jansepar talked with Liz about this. Can we just have the generator as the last step? I guess we still need this documented but most people will use the generator and so we could automate that step also.